CMakeLists.txt

```diff
@@ -14,6 +14,9 @@ cmake_minimum_required ( VERSION 2.8.8 FATAL_ERROR )
 # Use MSVS folders to organize projects on windows
 set_property(GLOBAL PROPERTY USE_FOLDERS ON)
 
+option(JSON_FORTRAN_USE_OpenCoarrays
+  "Build VTKmofo with support for linking against OpenCoarray programs" OFF)
```

Comment:
VTKmofo?

Reply:
Derp.... I saw that and meant to fix it but then lost track of it. (https://github.com/porteri/vtkmofo)